symforce/codegen/codegen_util.py

```diff
@@ -543,6 +543,13 @@ def _load_generated_package_internal(name: str, path: Path) -> T.Tuple[T.Any, T.
 def load_generated_package(name: str, path: T.Openable) -> T.Any:
     """
     Dynamically load generated package (or module).
+
+    name is the full name of the package or module to load (for example,
+    "pkg.sub_pkg" for a package called "sub_pkg" inside of another package
+    "pkg", or "pkg.sub_pkg.mod" for a module called "mod" inside of pkg.sub_pkg).
+
+    path is the path to the directory (or __init__.py) of the package, or the python
+    file of the module.
```

Comment:
Probably use the `Args:` style block since these are already describing the two args?

Reply:
Done